test/cmd/images_tests.sh

```diff
@@ -238,7 +238,8 @@ os::test::junit::declare_suite_start "cmd/images${IMAGES_TESTS_POSTFIX:-}/delete
 # test deleting a tag using oc delete
 os::cmd::expect_success_and_text "oc get is perl --template '{{(index .spec.tags 0).name}}'" '5.16'
 os::cmd::expect_success_and_text "oc get is perl --template '{{(index .status.tags 0).tag}}'" 'latest'
-os::cmd::expect_success "oc delete istag/perl:5.16 --context='${cluster_admin_context}'"
+project="$( oc project -q )"
```

Comment:
Normally it does not require, but the [user created for this test](https://github.com/openshift/origin/blob/c97d6e7562b53ab0cf09f51219614d30d52a0592/test/cmd/images-old-policy.sh#L27) has no such powers.